### ISSUE TITLE
Show the code host in the search results

### ISSUE THREAD

**Issue by mike-r-mclaughlin** [sourcegraph/sourcegraph#31878]
#### Feedback

When you have many code hosts, often of the same type (multiple GHE hosts and GH.com, for example), it's difficult to distinguish where the code lives from the search result without hovering over the link and looking at the result.  

It would be great to provide a configuration to include that in the breadcrumb.  Initially, we thought we could do it with the `repositoryPathPattern` in the code host configuration.  But, that does not display in the search results.

##### Direct quote
> while i was searching, it wasn’t readily obvious where the code lived - like, if it came from git.test vs Github, and if it was in git.test, which Org the code based was from.
Can we please have the config option to add the code host in the breadcrumbs in the search results, ex: `Git.test > gus/guscore > Jenginsfile` in this case?

#### Customer

[Customer](https://github.com/sourcegraph/accounts/issues/6716)
[Slack discussion](https://sourcegraph.slack.com/archives/C01P4R0RZH9/p1645731324111559)

**Comment by benvenker** [comment]
Interesting idea. @limitedmage this sounds like it does fall on us, do you agree?

**Comment by limitedmage** [comment]
Yes, this is ours. Looks related to #20874. It requires some refactoring of the data the server returns, since the server does not return the full repo name for content results.

**Comment by camdencheek** [comment]
This is still an issue. We no longer use pre-formatted headers, and for code hosts that are unknown to us (i.e. not github.com, bitbucket.com, or gitlab.org), we do not even render a code host icon that can be hovered over.

![Image](https://github.com/sourcegraph/sourcegraph/assets/12631702/579f64d1-f335-4079-ba75-7f3188cac37f)

cc @taiyab for design
